frontend_server/lib/server.dart

```diff
@@ -182,10 +182,13 @@ class _FrontendCompiler implements CompilerInterface {
     final String boundaryKey = new Uuid().generateV4();
     _outputStream.writeln('result $boundaryKey');
     final Uri sdkRoot = _ensureFolderPath(options['sdk-root']);
+    final String platformKernelDill =
+        options['strong'] ? 'platform_strong.dill' : 'platform.dill';
     final CompilerOptions compilerOptions = new CompilerOptions()
       ..sdkRoot = sdkRoot
       ..packagesFileUri = options['packages'] != null ? Uri.base.resolveUri(new Uri.file(options['packages'])) : null
       ..strongMode = options['strong']
+      ..sdkSummary = sdkRoot.resolve(platformKernelDill)
```

Comment:
@jensjoha , do we still need to specify `sdkRoot`? Now that we pass `sdkSummary` explicitly pointing to the platform dill file, `sdkRoot` seems redundant.

Reply:
I'm not sure. Maybe @peter-ahe-google knows.